crowdsourcing/models.py

```diff
@@ -274,7 +274,7 @@ class TemplateItem(models.Model):
     data_source = models.CharField(max_length=256, null=True)
     layout = models.CharField(max_length=16, default='column')
     type = models.CharField(max_length=16)
-    sub_type = models.CharField(max_length=16)
```

Comment:
we want to keep subtype, will be useful for example for groups of items e.g type: input sub_type: number, but additional changes need to be made so it's fine for now